### DISCUSSION TITLE
It looks weird to add a border color of width 2 in `CTkScrollableFrame`.

### DISCUSSION THREAD

**Discussion by elly-hacen** [TomSchimansky/CustomTkinter#1833] [General]
The first box is a `CTkFrame`, and the second one is a `CTkScrollableFrame`. However, when I added the border color, the `CTkFrame` looks pretty, but the `CTkScrollableFrame` does not look as intended. 

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/9669f383-1e48-4484-a3e8-aa090b9b2510)

```
class ChatGptResponseFrame(CTkScrollableFrame):
    def __init__(self, master, width, height, win_copilot_instance, **kwargs):
        super().__init__(master, **kwargs)

        self.win_copilot = win_copilot_instance

        self.configure(width=width, height=height,
                       corner_radius=25, fg_color='#222327',
                       border_color='#e9eef6', border_width=2)
```
```
class UserInterface(CTkFrame):
    """
        Represents the user interface for Windows Copilot.
    """

    def __init__(self, master, width, height, win_copilot_instance, **kwargs):
        super().__init__(master, **kwargs)

        self.win_copilot = win_copilot_instance

        self.configure(width=width, height=height,
                       corner_radius=25, fg_color='#222327',
                       border_color='#e9eef6', border_width=2)
```
```
class WinCopilot(CTk):
    def __init__(self):
        super().__init__()

        self.my_frame = UserInterface(self, width=400, height=300, win_copilot_instance=self)
        self.my_frame.place(relx=0.3, rely=0.45, anchor="center")

        self.scrollable_frame = ChatGptResponseFrame(self, width=370, height=250, win_copilot_instance=self)
        self.scrollable_frame.place(relx=0.75, rely=0.45, anchor="center")
```

**Comment by avalon60** [comment]
What did you intend?

**Reply by elly-hacen** [reply]
I was trying to ensure that the border color formed a perfect square and that the scroll bar was contained within the border. However, I encountered some difficulties achieving this.

**Reply by avalon60** [reply]
I've always assumed that that was the way it was meant to look, but I see your point. Why not raise an issue, and see what comes back?

**Reply by ElectricCandlelight** [reply]
Why not just place it inside a frame?

**Reply by elly-hacen** [reply]
Thank you for the idea! I'll give placing the elements inside a frame a try. It might help achieve the desired appearance. If the results are not as I want them to be, then I'll go ahead and raise an issue 🚀 to get some feedback on the design. 🙂

**Reply by avalon60** [reply]
A good idea, I guess you'll need to set the CTkScrollableFrame border width to zero.

**Reply by elly-hacen** [reply]
No, I can't set border width to zero, because I want border width. 😶

**Comment by elly-hacen** [comment]
# _Here, I've got some interesting results! 🙂_

### ```Dark Mode + Border Width = 5 (border_color='Green')```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/1d0ce48a-f587-466e-a866-ebfff1796e79)

### ```Light Mode + Border Width = 5 (border_color='Green')```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/3844d727-1566-43dc-bf5c-e2c191711c51)

### ```Light Mode + Border Width = 5 (border_color='#222327')```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/5ba39fcc-10c0-404c-a41d-38e69e23585a)

### ```Dark Mode + Border Width = 5 (border_color='#222327')```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/ed344366-ebd5-4a78-a4a6-e969d7e6eb05)

### ```Light Mode + Border Width = 5 (border_color='white')```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/c5085ebe-458a-4fe9-b827-e27369c1a12c)

### ```Light Mode With no Border```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/6cbf1a43-17e2-4114-85d6-5fc0ab6be92f)

### ```Light Mode With fg_color='white'```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/ccb43b3e-8fdb-4eb4-9672-3843676d64d6)

### ```Light Mode With fg_color='#222327'```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/b2f20128-4313-4803-b3de-5561b4d84c30)

### ```Light Mode With border_color='yellow', border_width=5```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/112799156/ff6a3707-aa28-4cd5-a610-7f340feffc70)
